pkg/purl/purl.go

```diff
@@ -407,6 +411,18 @@ func parseNpm(pkgName string) (string, string) {
 	return parsePkgName(name)
 }
 
+// ref. https://github.com/package-url/purl-spec/blob/7759d1cf81629267742eeeb0cdfccf5ebd624cc5/PURL-TYPES.rst#julia
+func parseJulia(pkgName string, pkgUUID string) (string, string, packageurl.Qualifiers) {
+	namespace, name := parsePkgName(pkgName)
+	qualifiers := packageurl.Qualifiers{
+		{
+			Key:   "uuid",
+			Value: pkgUUID,
+		},
+	}
+	return namespace, name, qualifiers
+}
+
```

Comment:
can you add testcase for this? (`TestNewPackageURL`)

Reply:
Done